### ISSUE TITLE
The SVG in the LuxDisclosure component should be treated as a decorative image

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/lux-design-system#210]
### Steps to recreate

1. Go to https://pulibrary.github.io/lux-design-system/
2. Scan the page with the axe devtools
3. Note that there is an error called `<svg> elements with an img role must have an alternative text`

### URL

* https://pulibrary.github.io/lux-design-system/
* https://allsearch.princeton.edu/?q=broccoli (next to the Show abstract)

### Issues

The LuxDisclosure component has a little SVG icon to give sighted users an extra hint that you can click on it.  However, since a screen reader already will announce it as a button with the text provided via props, the SVG should be hidden with aria-hidden=true.

### Severity

Low - Voiceover and NVDA just don't announce the svg, which is correct.

### WCAG Conformance level (A, AA, AAA)

Level A

### User impact

Low.

### Recommended fix

Change aria-hidden=false to true.

### Acceptance criteria
- [ ] Axe devtools does not complain about this SVG
